app/src/main/res/layout/dialog_fragment.xml

```diff
@@ -20,6 +20,8 @@
         android:id="@+id/input_dialog_fragment"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:ems="3"
```

Comment:
what does this do?

Reply:
It sets the width of the TextView to fit a text

Reply:
is it necessary?

Reply:
okay